### PR TITLE
Add telemetry publishing support

### PR DESCRIPTION
**Pull request by aestene**

### CODE REVIEW

File: src/isar_robot/robotinterface.py [hunk 1]
```diff
@@ -65,3 +74,46 @@ def get_inspections(self, step: InspectionStep) -> Sequence[Inspection]:
...
+        battery_thread: Thread = Thread(
+            target=battery_publisher.run,
+            name="ISAR Robot Battery Publisher",
+            daemon=True,
+        )
+        publisher_threads.append(battery_thread)
+
+        return publisher_threads
+
+    def _get_pose(self) -> str:
```
[the remark this answers is not in the document]
Comment (aestene) [reply src/isar_robot/robotinterface.py:109]: While I wouldn't do it for the other repositories I think it's fine here until it grows too large. They are marked as internal through the naming convention and it is best to keep this repository as simple as possible.

**Review by oysand — CHANGES_REQUESTED** [review]
See comment

File: src/isar_robot/robotinterface.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (oysand) [reply src/isar_robot/robotinterface.py:109]: Agreed

**Review by oysand — APPROVED** [review]
LGTM

**Pull request merged**